### PR TITLE
Add prettier to codebase (round 2)

### PR DESCRIPTION
**Pull request by blainekasten**
Let's prettify this codebase! 💅 

This PR does a few thing, but i'll highlight them here to simplify the review.

1. 245 files are changed _because_ we are running prettier for the first time.
1. We use `eslint-config-prettier` to turn off prettier rules
1. `.prettierrc` is used to set `arrowParens` to `always`
1. Add `format.ci` to the `check.ci` to check in CI if files are not prettier compatible.
1. Add a command: `yarn nps format` to run the formatting on all files

### CODE REVIEW

**Comment by ryan-roemer** [comment]
Nice work so far! I'd suggest a script that fails CI like we have for spectacle

https://github.com/FormidableLabs/spectacle/blob/master/package.json#L27-L29

```js
    "prettier": "prettier \"**/*.{js,json,css,md}\"",
    "prettier-fix": "npm run prettier -- --write",
    "prettier-check": "npm run prettier -- --list-different",
```

From there `yarn prettier-check` will exit non-zero if any files would be changed and is part of spectacle's `check-ci` script running in CI.

File: test/client/spec/victory-core/victory-util/data.spec.js [hunk 1]
```diff
@@ -141,8 +137,8 @@ describe("victory-util/data", () => {
           const data = createData([{ x: "kittens", y: 3 }, { x: "cats", y: 5 }]);
           const props = { data, x: "x", y: "y" };
           const expectedReturnWithEventKeys = [
-             { _x: 1, x: "kittens", xName: "kittens", _y: 3, y: 3 },
-             { _x: 2, x: "cats", xName: "cats", _y: 5, y: 5 }
+            { _x: 1, x: "kittens", xName: "kittens", _y: 3, y: 3, eventKey: 0 },
```
Comment (boygirl) [inline test/client/spec/victory-core/victory-util/data.spec.js]: These `eventKey` properties just got removed, which is why your specs are failing here

File: test/client/spec/victory-core/victory-util/data.spec.js [hunk 2]
```diff
@@ -218,10 +214,9 @@ describe("victory-util/data", () => {
           const returnData = Data.getData({ data, sortKey: "order", sortOrder: "descending" });
 
           expect(returnData).to.eql([
-            { _x: 2, x: 2, _y: 2, y: 2, order: 3 },
-            { _x: 1, x: 1, _y: 1, y: 1, order: 2 },
-            { _x: 3, x: 3, _y: 3, y: 3, order: 1 }
-
+            { _x: 2, x: 2, _y: 2, y: 2, order: 3, eventKey: 0 },
```
Comment (boygirl) [inline test/client/spec/victory-core/victory-util/data.spec.js]: ditto

**Comment by blainekasten** [comment]
@ryan-roemer I basically already have that. Victory uses the package-scripts.json to list it's scripts. I currently have the equivalent of `prettier-fix` and `prettier-check`. I'm not sure if I see the need for the dry run version?

**Comment by ryan-roemer** [comment]
Ah, yeah, missed `format.ci` in package-scripts. Looks good!

**Pull request merged**